accepted/0000-governance.md

```diff
@@ -0,0 +1,65 @@
+# Paketo Governance RFC
+
+This RFC outlines the govornance structure for the Paketo project.
+
+## Organizations:
+
+Paketo consists of two github organizations:
+
+- `paketo-buildpacks`
+- `paketo-community`
+
+The `paketo-buildpacks` github organization consists of actively supported buildpacks, libraries etc.
+
+The `paketo-community` github organization is designed as a staging ground for acceptance into the offically supported `paketo-buildpacks` org. Promotion from `paketo-community` into `paketo-buildpacks` requires an official [`rfc`](github.com/paketo-buildpacks/rfcs)
+
+## Contribution Model:
+
+The paketo contribution model is divided into the following, Subteams & Roles.
+
+- A subteam is a group responsible some set of repositories in the `paketo-buildpacks` org. This limits the repositories any individual is allowed to take action in.
+
+- Roles are Contributor, Maintainer, And Steering Commitee member, this limits what kinds of actions an individual may be allowed to make in the `paketo-buildpacks` org
+
+### Subteams
+Subteams are responsible for maintaining some set of repositories under the `paketo-buildpacks` org. A natural split is into "Language Teams" responsible for maintaining all buildpack & specification changes that pertain to a particular language ecosystem.
+
+In order to become a subteam member a user must be a contributor.
+
+Subteams have ownership & voting power over relevant `rfcs`.
+
+Both maintainers contributors & core-team members can be members on any number of Subteams.
```

Comment:
Is this meant to include all three of: maintainers, contributors, and core-team members?